### DISCUSSION TITLE
Search bar

### DISCUSSION THREAD

**Discussion by aolko** [Moonvy/figmaEX#4] [Ideas]
Please add a search bar to search for plugins

**Comment by yArna** [comment]
Is this what you want?
[Quick Actions](https://help.figma.com/hc/en-us/articles/360040328653-Use-shortcuts-and-quick-actions)
![](https://help.figma.com/hc/article_attachments/1500003682141/Quick_actions_bar_in_the_Editor__1_.png)

**Reply by aolko** [reply]
no, i'd like the search in figmaex directly

**Reply by NikolayGur** [reply]
Yes it would be convenient. For example to add plugins to the top of the panel, or just start. But I would also suggest to implement the ability to quickly delete plugins from the panel and add them. Exactly from the panel and in the panel, so that the plugins would remain installed in the figma itself. And the panel would be a place for quick access to favorite plugins. Which you could change as needed.
